### DISCUSSION TITLE
Host paths are passed as parameters to Docker Actions executed from PHPStorm UI

### DISCUSSION THREAD

**Discussion by bserem** [php-perfect/ddev-intellij-plugin#151] [Ideas]
I'll try to explain this to the best of my abilities. Hopefully the attached image will help more than my English skills here.

When I try to run a task (`robo` in my case) from the UI by pressing the green play icon docker (ddev) runs properly, but the argument passed to the action is using a host path, rather than a ddev-related path.

![image](https://user-images.githubusercontent.com/520237/214049368-30881fa9-8b90-4333-94f5-a7eaa7919d47.png)

It seems something related to ddev-integration plugin. If you feel it is not please ignore me and close it.

Thanks (and even more thanks for this plugin)

**Comment by nico-loeber** [comment]
Hey @bserem,

sorry for my late reply.

I don't know robo and I don't know what this play button is executing.
I see the output on the terminal but this looks like some docker compose based task and not a ddev action.

What would you expect to happen and how can the ddev integration plugin help you here?

**Comment by bserem** [comment]
Hey @nico-loeber!

**Short intro to Robo**
Robo is a php task runner (https://robo.li/). It is a requirement for Drush and thus it exists in almost all Drupal installations. It can be used for many things. I personally use it to keep the same commands (eg: `theme:compile`) across different project, regardless of what needs to be done in the background. So, a developer only needs to run `robo theme:compile` and not worry about anything not in his jurisdiction.

**PHPStrom & Robo**
As of phpstorm 2022.3, Robo task are identified and can be called from the UI (https://blog.jetbrains.com/phpstorm/2022/12/phpstorm-2022-3-whats-new/#robo_integration). This came as I surprise to me, I missed the update but saw the green "play" buttons in the IDE.
This has the extra bonus that robo tasks can be added in README.md files and be executed from inside the MD! (screenshot in first comment) which imho can facilitate onboarding of users to a project.

**The actual issue**
You are correct. When I click the "play" button a docker-compose task is being executed. For reasons I do not know, the `--load-from` argument is from my host machine and not from the docker/ddev virtual machine.
I suppose this happens for other actions too, but I am not using them (yet?).

I thought that these arguments were handled by the integration. My mistake.
If you feel that this is something that can be added in the plugin it would be great. In any case, feel free to close the issue.

Thanks
